### ISSUE TITLE
Operand over 16-bits. Got 72000.

### ISSUE THREAD

**Issue by Turbo87** [emberjs/ember.js#16269]
Using the latest 3.1 beta version I'm seeing an error "Operand over 16-bits. Got 72000." in one of our tests which does this:

```
render(hbs`{{some-component foo=9000}}`);
```

It seems that the maximum value for literal numbers has decreased significantly in the 3.1-beta 🤔 

https://github.com/emberjs/ember.js/issues/15635 might also be related to this.

**Comment by rwjblue** [comment]
FWIW - This is ultimately a glimmer-vm issue...

**Comment by Turbo87** [comment]
Still needs release and update here, right?

**Comment by rwjblue** [comment]
Yes, I didn’t close this intentionally...

**Comment by rwjblue** [comment]
Fixed by https://github.com/emberjs/ember.js/pull/16286

**Comment by jdurand** [comment]
I opened an issue https://github.com/cibernox/ember-ast-helpers/issues/23 but I'm going to log this here just in case this is a regression.

I started having these issues when I upgraded my app to ember-source@3.3.2 (and ember-data@3.4.0 mentioning because from the little I could understand one side effect could be broken hasMany relationships)

Here's a production stack trace: https://sentry.io/share/issue/b3223fa152b04552bf19ab519fcd2218/

I haven't seen this error yet for users that I rolled back to ember-source@3.1.4 & ember-data@3.1.2. I'll report back if I see something else.
